examples/cpu/fractal.cpp

```diff
@@ -42,9 +42,9 @@ int main(void)
      * it can be used for rendering text */
     fg::Font fnt;
 #ifdef OS_WIN
-    fnt.loadSystemFont("Calibri", 32);
+    fnt.loadSystemFont("Calibri");
 #else
-    fnt.loadSystemFont("Vera", 32);
+    fnt.loadSystemFont("Vera");
 #endif
```

Comment:
No, we are not currently including these. Vera family is one of the most common font families available on Unix/Linux systems. Similarly, Calibri on Windows. I don't think we need to include them unless there is a possibility of these fonts missing on these systems.